.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📄 Blank Issue
+    url: https://github.com/openxla/stablehlo/issues/new
+    about: If you know what you're doing (especially collaborators)
+  - name: 🗣 Start a discussion on GitHub
+    url: https://github.com/openxla/stablehlo/discussions
+    about: An alternative platform for asynchronous discussion
```

Comment:
"For asynchronous discussion"? We currently have only one platform for these discussions.

Reply:
Updated to "For early feedback on a design or other proposal"

Will reach out to discuss if this accurately captures the role of GH discussions before marking as resolved.

Reply:
Thank you! This is a great phrasing.